install/share/expire_msg.template

```diff
@@ -0,0 +1,5 @@
+Hi {{ fullname }},
```

Comment:
How about your turn the template into a RFC 822 message and parse the output of JINJA with [email.message_from_string](https://docs.python.org/3/library/email.parser.html?highlight=message_from_string#email.message_from_string)?

This would allow an admin to specify email headers including subject or custom headers in the template. I did a quick test. The parser automatically converts headers with non-ASCII char.

```
>>> text = """\
... Subject: Blä
... 
... Test äöü
... """
>>> print(email.message_from_string(text).as_string())
Subject: =?utf-8?q?Bl=C3=A4?=

Test äöü
```

Reply:
In the future EPN could also look at the language and locale attribute of the user and choose a language specific template (e.g. ``expire_msg.fr.template``). With my proposal you have the subject and body in one file.

Reply:
I agree that it'd be nice to have a configurable subject. Francois took the approach of having the body of the message be an attachment so the encoding is automatic. Arguably this will make it difficult for text-based readers. He did want to allow html-based e-mails which is why there is the content subtype (defaults to plain). We could add subject as an option in the config file, and may allso allow X- headers to be defined.

I'm not sure about separate templates. We don't have information on the target other than what is in LDAP and we don't currently maintain language.